src/Grpc/Server.php

```
@@ -138,15 +149,56 @@ public function serve(?WorkerInterface $worker = null, ?callable $finalize = nul
         }
 
-        return $this->services[$service]->invoke($method, $context, $body);
+
+        $handler = new class($this->services[$service]) implements HandlerInterface {
+            public function __construct(private $service) {}
+            public function handle(\Spiral\Interceptors\Context\CallContextInterface $context): mixed
+            {
+                $target = $context->getTarget();
+                $args   = $context->getArguments();
+
+                [$serviceName, $method] = explode('/', (string) $target, 2);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$serviceName'. (undefined)

(UnusedLocalVariable)